src/stores/contactmoment/index.ts

```diff
@@ -11,35 +11,66 @@ import { defineStore } from "pinia";
 import { createSession, type Session } from "../switchable-store";
 export * from "./types";
 import type { ContactVerzoekVragenSet } from "@/features/contactverzoek/formulier/types";
-import type { Afdeling } from "@/composables/afdelingen";
 export type ContactmomentZaak = { zaak: ZaakDetails; shouldStore: boolean };
 
-export interface ContactverzoekAfdeling {
+export interface Afdeling {
```

Comment:
ContactverzoekAfdeling is hernoemd naar Afdeling. Er werd al een type Afdeling gebruikt. Deze werd geïmporteerd uit "@/composables/afdelingen". Die werd alleen gebruikt voor de afdeling van een contactmoment, maar daar slaan we alleen een naam van op. Ik vraag me daarom af waarom in de interface Vraag het type van de Afdeling  is veranderd van string naar Afdeling ( 9/6/2023 commit 7194f099) @felixcicatt kan jij je dat herinneren?

Reply:
Dat weet ik niet meer precies. Ik kan me voorstellen dat het me stabieler leek om de hele afdeling bij te houden, voor het geval je op een andere plek niet alleen de naam maar ook de id nodig hebt. Dit was volgens mij in de context van een story om de afdeling van een kennisartikel te gebruiken om de afdeling bij een contactmoment voor in te vullen. De conclusie was uiteindelijk dat er geen garantie is dat de afdelingen uit de bron van de kennisartikelen dezelfde afdelingen zijn als die van de medewerkers, dus wellicht dat het daardoor niet meer relevant was, maar misschien wist ik dat op het moment van die commit nog niet.